tutormfe/templates/mfe/build/mfe/Dockerfile

```diff
@@ -65,12 +47,11 @@ RUN --mount=type=cache,target=/root/.npm,sharing=shared npm clean-install --no-a
 {{ patch("mfe-dockerfile-post-npm-install") }}
 {{ patch("mfe-dockerfile-post-npm-install-{}".format(app_name)) }}
 COPY --from={{ app_name }}-src / /openedx/app
-COPY --from={{ app_name }}-i18n /openedx/app/src/i18n/messages /openedx/app/src/i18n/messages
 
 # Whenever a new MFE supports Atlas, it should be added to this list.
 # When all MFEs support Atlas, this if-statement should be removed.
 {% if app_name in ["communications"] %}
-RUN make OPENEDX_ATLAS_PULL=true pull_translations
+RUN make OPENEDX_ATLAS_PULL=true ATLAS_OPTIONS="--repository={{ ATLAS_REPOSITORY }} --branch={{ ATLAS_REVISION }} {{ ATLAS_OPTIONS }}" pull_translations
```

Comment:
> now that we're moving to using revisions (tags/shas) instead of branches when pulling with atlas, does it make sense to still use --branch here?

@brian-smith-tcril the `ATLAS_REVISION` choice is a bit debatable but I've accepted [Regis feedback](https://github.com/overhangio/tutor/pull/964#discussion_r1429673724), but we ended up using it for Tutor even though it's still technically the same `--branch` argument.

Currently the `--branch` argument supports both tags and branches, but not commit SHA due to `git clone --branch` limitation:

```
$ git clone --help | grep -A2 '.-branch '
       -b <name>, --branch <name>
           Instead of pointing the newly created HEAD to the branch pointed to by the cloned repository’s HEAD, point to <name> branch
           instead. In a non-bare repository, this is the branch that will be checked out.  --branch can also take tags and detaches the
           HEAD at that commit in the resulting repository.
```

We haven't added support yet for all git revisions:

 - https://github.com/openedx/openedx-atlas/issues/48

> If not, it might make sense to update the atlas CLI to use a new argument name before merging this.

In princinple I agree that adding the `--revision` option is better before merging this pull request, but it'll require a backlog of small pull requests to be merged across multiple repositories which could take a significant time.

Once this is done, I believe we can make it backward compatible way by adding a new `--revision` option which uses `--branch` as an alias.

Therefore, I think keeping `ATLAS_REVISION` is somewhat acceptable even though it doesn't match `atlas pull --branch` name.

To summarize, if adding the `--revision` option to `atlas` is a priority, I believe it should be done after merging this pull request since we're almost done and I'd like to keep a good window with Atlas being merged before Redwood is realesed.

Reply:
Are we using `--branch` anywhere else yet? If not, could we just make a quick PR to `atlas` changing the current `--branch` argument to `--revision` and update it to support pulling by `sha` after?

Reply:
@brian-smith-tcril The rename has been done. Thanks for bringing it up.


@arbrandes This is now ready for merge.